pages/docs/reference/js-overview.md

```diff
@@ -5,52 +5,51 @@ category: "Introduction"
 
-    * **Interacting with graphics such as WebGL**. You can use Kotlin to create graphical elements on a web page using WebGL.
-
-* Creating Kotlin code that targets server-side JavaScript
-
-    * **Working with server-side technology**. You can use Kotlin to interact with server-side JavaScript such as Node.js
-
-Kotlin can be used together with existing third-party libraries and frameworks, such as jQuery or React. To access third-party frameworks
-with a strongly-typed API, you can convert TypeScript definitions from the [Definitely Typed](http://definitelytyped.org/)
-type definitions repository to Kotlin using the [dukat](https://github.com/kotlin/dukat) tool. Alternatively, you can use
-the [dynamic type](dynamic-type.html) to access any framework without strong typing.
-
-Kotlin is compatible with CommonJS, AMD and UMD, [making interaction with different](/docs/tutorials/javascript/working-with-modules/working-with-modules.html) module systems straightforward.
+Kotlin/JS is also compatible with the most common module systems: UMD, CommonJS, and AMD. Being able to [produce and consume modules](/docs/tutorials/javascript/working-with-modules/working-with-modules.html) means that you can interact with the JavaScript ecosystem in a structured manner.
 
 ## Kotlin/JS, Today and Tomorrow
 
-Want to know more about Kotlin/JS? 
+**Want to know more about Kotlin/JS?**
 
-In this short video, Kotlin Developer Advocate Sebastian Aigner will explain the main Kotlin/JS benefits to you, share some tips and use cases, and also tell you about the plans and upcoming features for Kotlin/JS.
+In this video, Kotlin Developer Advocate Sebastian Aigner will explain the main Kotlin/JS benefits to you, share some tips and use cases, and also tell you about the plans and upcoming features for Kotlin/JS.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/fZUL8_kgHXg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 
 ## Getting Started with Kotlin/JS
 
-To find out how to start using Kotlin for JavaScript, please refer to the [Setting up a Kotlin/JS project](/docs/reference/js-project-setup.html).
+To find out how to start using Kotlin for JavaScript, please refer to the [Setting up a Kotlin/JS project](/docs/reference/js-project-setup.html), or pick a hands-on lab from the next section to work through.
```

Comment:
As one option to get started, we can add a link to JS tutorial for React.

Reply:
Also on other overview pages, we have some kind of "New to Kotlin?" question that recommends  _Basic syntax_. I don't know how it's relevant for JS.